.github/workflows/main.yml

```diff
@@ -19,7 +19,16 @@ jobs:
         run: |
           python -m pip install tox
       - name: Run tests
-        run: tox
+        run: tox -- --junit-xml=.test-results/pytest/cov.xml
```

Comment:
Does codecov use junitxml? I'm pretty sure junitxml is for test results and not coverage results. I suspect this setting is unnecessary and instead we need a setting to cause coverage results to be exported to an xml file.

Reply:
Fixed in 725be0142.

Reply:
Isn't it the same?

Reply:
Oh, I confused test results with coverage